content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,443 @@ categories:
+| `EF_RUNTIME`    | The maximum number of top candidates to hold during the KNN search. Higher values lead to more accurate results at the expense of a longer query runtime. | Value passed during index creation (default is 10). |
+| `EPSILON`       | Relative factor that sets the boundaries for a vector range query. Vector candidates whose distance from the query vector is `radius * (1 + EPSILON)` are potentially scanned, allowing a more extensive search and more accurate results at the expense of runtime. | Value passed during index creation (default is 0.01). |
+
+
+
+### Important notes
 
 {{% alert title="Important notes" color="info" %}}
 
-1. Although specifying `K` requested results in KNN search, the default `LIMIT` is 10, to get all the returned results, specify `LIMIT 0 <K>` in your command.
+1. Although specifying `<top_k>` requested results in KNN search, the default `LIMIT` is 10, to get all the returned results, specify `LIMIT 0 <top_k>` in your search command. See examples below.
 
-2. By default, the results are sorted by their document's score. To sort by some vector similarity score, use `SORTBY <dist_field_name>`. See examples below.
+2. By default, the results are sorted by their document's score. To sort by some vector similarity score, use `SORTBY <distance_field>`. See examples below.
 
-3. It is recommended to adjust the `<radius>` parameter in range queries to the corresponding vector field distance metric and to the data itself. In particular, recall that the distance between the vectors in an index whose distance metric is Cosine is bounded by `2`, while L2 distance between the vectors is not bounded. Hence, it is better to consider the distance between the vectors that are considered similar and choose `<radius>` accordingly.   
+3. It is recommended to adjust the `<radius>` parameter in range queries to the corresponding vector field distance metric and to the data itself. In particular, recall that the distance between the vectors in an index whose distance metric is Cosine is bounded by `2`, while L2 distance between the vectors is not bounded. Hence, it is better to consider the distance between the vectors that are considered similar and choose `<radius>` accordingly.
 
 {{% /alert %}}
 
+
 ## Vector search examples
 
-### "Pure" KNN queries
-Return the 10 documents for which the vector stored under its `vec` field is the closest to the vector represented by the following 4-bytes blob:
+Below are a number of examples to help you get started. If you want a more comprehensive walkthrough, visit the vector [quickstart guide]({{< baseurl >}}/develop/get-started/vector-database) and the [Redis AI Resources](https://github.com/redis-developer/redis-ai-resources) repo for more assistance.
+
+### Standard vector search examples
+Return the 10 nearest neighbor documents for which the vector stored under its `vector` field is the closest to the vector represented by the following 4-bytes blob:
```

Comment:
Again, let's not call the vector field "vector". Makes the sentence very confusing to read.

Reply:
Better: The query below returns the 10 documents whose "message_embedding" vector fields most closely match the provided 4-byte query vector. Here, the query vector is the byte string "\x12\xa9\xf5\x6c".

Of course, this requires renaming the vector field in the example.

Reply:
Point heard, making the update.